app/models/bookmark.rb

```diff
@@ -91,6 +87,10 @@ def auto_delete_on_owner_reply?
     self.auto_delete_preference == Bookmark.auto_delete_preferences[:on_owner_reply]
   end
 
+  def reminder_at_ics(offset: nil)
+    (offset.present? ? reminder_at + offset : reminder_at).strftime(I18n.t("datetime_formats.formats.calendar_ics"))
+  end
```

Comment:
```suggestion
  def reminder_at_ics(offset: 0)
    (reminder_at + offset).strftime(I18n.t("datetime_formats.formats.calendar_ics"))
  end
```
How about this instead? I don't think we expect anyone to pass in a `nil`?